Compress.cpp

```diff
@@ -71,6 +72,7 @@ const char* usage_Compress =
 "                                  in it, i.e. no .inf file is created\n"
 "                                  [default: bin1]\n"
 "  -o BINFILE, --output=BINFILE    filename used to write the produced automaton\n"
+"  -p, --pack_inp                  create packed binary .inp file instead .inf\n"
```

Comment:
Instead of `create packed binary .inp file instead .inf,` I suggest `Create a packed inf file (.inp)`